### ISSUE TITLE
Error when running WAN setting experiment

### ISSUE THREAD

**Issue by HuangPZ** [LaRiffle/ariann#3]
Hi! I was trying to run the WAN setting by installing from the source. I followed all instructions except two things: 1. I created a python venv for the project and 2. I put all folders into ~/work instead of in ~. I've changed the home dir in the .sh files accordingly.

Without adding the --websocket flag, the experiment runs normally. However if with it, I got the following error saying "Connection refused". 
![image](https://user-images.githubusercontent.com/28959049/157747780-2223fdc0-c150-4c0d-b746-bff5d9325842.png)

Do I have to do something in advance to initialize the port? The firewall on the server should be closed.

**Comment by LaRiffle** [comment]
This is very strange... You have done all the fishy steps provided in https://github.com/LaRiffle/ariann#2-pygrid-for-cross-node-execution-only ?
You are on a single machine ?

**Comment by HuangPZ** [comment]
Yes. They are all in a single machine. As I said the only diff is the folder location and virtual env

**Comment by HuangPZ** [comment]
Hi, I'm still having this problem. Any clue on how to solve it? I guess you didn't see such a problem on your machine ofc. 
I had to change the install location due to limited space in ~. But otherwise I should have done every step.
